.gitignore

```diff
@@ -79,3 +79,6 @@ keys/
 
 # clangd language server
 .clangd/
+
+# ignore external packages
+packages/
```

Comment:
could we move this to build/pkg? otherwise, we'll have "packages" and "pkg" in the root.

Reply:
Yep, this was part of the discussion. "Where to store the package sources". I'm fine with the proposal.